lizmap/modules/admin/controllers/theme.classic.php

```diff
@@ -65,6 +70,14 @@ public function modify()
         /** @var jResponseRedirect $rep */
         $rep = $this->getResponse('redirect');
 
+        // Redirect to the theme home page if the user has not enough rights
+        if (!\jAcl2::check('lizmap.admin.theme.update')) {
```

Comment:
Because of the setting of `$pluginParams`, you don't need this test. The method will not even be executed, and the user will have a 403 error. If you don't want this error page, be sure there are no links to this action. So in any templates, you should check the right before display the link or button to this action.

Same comment for other methods.